provisioning/provisioning-service-client/pom.xml

```diff
@@ -43,6 +43,12 @@
             <artifactId>${iot-deps-artifact-id}</artifactId>
             <version>${iot-deps-version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>javax.json</artifactId>
+            <version>1.0.4</version>
+            <type>jar</type>
+        </dependency>
```

Comment:
Iot service client got this dependency through deps before, but now that deps does not depend on glassfish, we need to declare it here